[USING] rework Cold Wasabi (#734)
zkSNACKs/WasabiDoc: docs/using-wasabi/ColdWasabi.md

## What is Cold Storage

> Cold storage refers to keeping bitcoin protected by private keys which are generated and stored completely offline.

Reviewer: ```suggestion
Cold storage refers to generating and storing private keys completely offline.
```

This is an often-used security precaution, especially dealing with large amounts of bitcoin.
Because the private keys are not on a computer which is connected to the internet, many remote attack vectors are nullified.

Methods of cold storage include keeping private keys on a:
- USB drive or other data storage medium
- Paper wallet
- Bearer item such as a physical bitcoin
- Hardware wallet

## Hardware Wallet with Wasabi

You can use Wasabi Wallet with almost any hardware wallet out there, because Wasabi utilizes [Bitcoin Core Hardware Wallet Interface [HWI]](https://github.com/bitcoin-core/hwi).

> The setup is thoroughly tested for [Trezor model One and T](https://trezor.io), [Ledger Nano S](https://ledgerwallet.com), and [Coldcard](https://coldcardwallet.com).

Reviewer 2: ```suggestion
The setup is thoroughly tested for [Trezor model One and T](https://trezor.io), [Ledger Nano S](https://ledger.com), and [Coldcard](https://coldcardwallet.com).
```

### Connecting via USB

#### Import the wallet

When Wasabi is running, the hardware wallet can be connected via USB to the computer.
Wasabi should automatically detect the hardware, and open the `Hardware Wallet` tab where you can load the wallet.

> This wallet can be used as a watch-only wallet when the hardware wallet device is not connected, then it can be loaded as any other wallet in the `Wallet Explorer`.

Reviewer: > then it can be loaded as any other wallet in the `Wallet Explorer`

This means exactly the same as 

> This wallet can be used as a watch-only wallet when the hardware wallet device is not connected

```suggestion
This wallet can be used as a watch-only wallet when the hardware wallet device is not connected.
```
